homeconnect/api.py

```diff
@@ -31,10 +31,10 @@ def __init__(
         client_id: str = None,
         client_secret: str = None,
         redirect_uri: str = None,
-        api_url: str = None,
+        api_url: Optional[str] = URL_API,
         token_updater: Optional[Callable[[str], None]] = None,
     ):
-        self.host = api_url if api_url else URL_API
+        self.host = api_url
```

Comment:
`self.host = api_url or URL_API`, otherwise `self.host` can be `None`

Reply:
line 34，i give api_url default value with URL_API

Reply:
Oh right, great! But then the type hint should be `str` again as it can't be `None`

Reply:
ok, i think i should learn python typing deeper